README.md

```diff
@@ -16,32 +16,22 @@ nor to do a silly *fibonacci-as-a-service*, ok?
 
 ## What?
 
-The following microframeworks were considered when doing this research:
+Rack 2 compatible frameworks
 
-- [Brooklyn](https://github.com/luislavena/brooklyn) - [brooklyn.ru](apps/brooklyn.ru)
 - [Cuba](https://github.com/soveran/cuba) - [cuba.ru](apps/cuba.ru)
-- [Gin](https://github.com/jcasts/gin) - [gin.ru](apps/gin.ru)
+- [Flame](https://github.com/AlexWayfer/flame) - [flame.ru](apps/flame.ru)
 - [Grape](https://github.com/ruby-grape/grape) - [grape.ru](apps/grape.ru)
 - [Hanami (Router)](https://github.com/hanami/router) - [hanami-router.ru](apps/hanami-router.ru)
 - [Hobbit](https://github.com/patriciomacadden/hobbit) - [hobbit.ru](apps/hobbit.ru)
-- [Nancy](https://github.com/heapsource/nancy) - [nancy.ru](apps/nancy.ru)
-- [NYNY](https://github.com/alisnic/nyny) - [nyny.ru](apps/nyny.ru)
-- [Mustermann](https://github.com/rkh/mustermann) - [mustermann.ru](apps/mustermann.ru)
 - [Rack](https://github.com/rack/rack) - [rack.ru](apps/rack.ru) + [rack-response.ru](apps/rack-response.ru)
+- [Rack-App](https://github.com/rack-app/rack-app) - [rack-app.ru](apps/rack-app.ru)
 - [RackStep](https://github.com/mfdavid/rackstep) - [rackstep.ru](apps/rackstep.ru)
-- ~~[Rails](https://github.com/rails/rails) - [rails.ru](apps/rails.ru)~~
-- [Ramaze](https://github.com/Ramaze/ramaze) - [ramaze.ru](apps/ramaze.ru)
+- [Rails (API + Metal)](https://github.com/rails/rails) - [rails-api.ru](apps/rails-api.ru)  + [rails-metal.ru](apps/rails-metal.ru)
 - [Rambutan](https://github.com/NewRosies/rambutan) - [rambutan.ru](apps/rambutan.ru)
 - [Roda](https://github.com/jeremyevans/roda) - [roda.ru](apps/roda.ru)
-- [Scorched](https://github.com/Wardrop/Scorched) - [scorched.ru](apps/scorched.ru)
 - [Sinatra](https://github.com/sinatra/sinatra) - [sinatra.ru](apps/sinatra.ru)
 - [Syro](https://github.com/soveran/syro) - [syro](apps/syro.ru)
 
-Please note that while Rails has been added to the list, it is just a
```

Comment:
Because i change it to two Rails testing, first as Rails 5 API mode and second as pure Metal without with removed some middlewares.
But you are right. What is used is not whole Rails framework, but just ActionPack.